CS483/CS483/Kartaclysm/StateMachine/Gameplay/GameplayStatesEnum.h

```diff
@@ -0,0 +1,24 @@
+//========================================================================
+// GameplayStatesEnum
+// Author: David Hanna
+//
+// An enumeration to replace the int values in the state machine.
+//========================================================================
+
+#ifndef GAMEPLAY_STATES_ENUM
+#define GAMEPLAY_STATES_ENUM
+
+namespace Kartaclysm
+{
+	enum GameplayStates
```

Comment:
Consider putting this enum inside of another namespace as well, similar to InputActions.h, and maybe moving it into GamePlayState.h as they are tightly coupled.